src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  const store = {};
+
+  store.count = 0;
+
+  const funcObj = function(x) {
+    store.count++;
+
+    if (store.count >= 4 && store.count % 2 === 0) {
+      return () => 'Bzzz... Error!';
+    }
+
+    return (y) => x + y;
+  };
+
+  return funcObj;
```

Comment:
you can return this function right  away, no need for this variable
```suggestion
  return function(x) {
    store.count++;

    if (store.count >= 4 && store.count % 2 === 0) {
      return () => 'Bzzz... Error!';
    }

    return (y) => x + y;
  };
;
```

Reply:
fixed